indico/modules/rb_new/client/js/components/pages/RoomList.jsx

```diff
@@ -73,8 +72,26 @@ export default class RoomList extends React.Component {
         );
     };
 
+    closeBookingModal = () => {
+        const {pushState} = this.props;
+        pushState(`/rooms`, true);
```

Comment:
`'`

Reply:
I was looking for a rule but couldn't find one - just the [other way round](https://eslint.org/docs/rules/no-template-curly-in-string)